hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +125,128 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def test_wrap(s):
+
+    def generate_outside(box, interior_points, multipliers, initial_images):
```

Comment:
@cbkerr An improved method of organization would be to write these helper functions outside of the `test_wrap` function, and then have multiple `test_*` functions that call those for each case being tested.

So like:

```python
def generate_outside(...):
    # Implementation here
    pass

def run_box_type(...):
    # Implementation here
    pass

def test_cubic_box(s):
    run_box_type(...)

def test_triclinic_box(s):
    run_box_type(...)
```

This way each test appears as a separate entry in the pytest output.

Reply:
implemented this with keeping "wrap" in the name for running all those together `-k wrap`